versioned_docs/version-Jupiter/standards/CX-0135-CompanyCertificateManagement/CX-0135-CompanyCertificateManagement.md

```diff
@@ -166,6 +166,14 @@ The semantic model has the unique identifier
 
 > urn:samm:io.catenax.business_partner_certificate:1.0.0
 
+:::info
+The Aspect Model will be updated to version 3.0.0 within the next release (CX-Io):
```

Comment:
I'm not entirely sure if we'll deliver this exact data model with version 25.03, or if it might end up being a 3.1.0 data model instead, given that there are still so many months to go and the Expert Group is very active. For now, we would probably recommend starting with version 3.0.0.